demo/contracts/contracts/OracleInterfaces.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.16;
+
+enum FeedType {
+    DataStream,
+    PriceFeed
+}
+
+struct ForwardData {
+    int256 price;
+    FeedType feedType;
+    bytes forwardArguments;
+}
+
+interface IOracleConsumerContract {
```

Comment:
we probably will need to publish out contacts as an npm module and reuse interfaces instead of duplication.. but ok for demo

Reply:
exactly, it is a waste of time for demo